games-emulation/mednafen/mednafen-0.9.44.1.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools flag-o-matic pax-utils
+
+DESCRIPTION="Argument-driven multi-system emulator utilizing OpenGL and SDL"
+HOMEPAGE="https://mednafen.github.io/"
+SRC_URI="https://mednafen.github.io/releases/files/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="alsa altivec cjk debugger jack nls pax_kernel"
+
+RDEPEND="
+	dev-libs/libcdio
+	media-libs/libsdl[sound,joystick,opengl,video]
+	media-libs/libsndfile
+	sys-libs/zlib[minizip]
+	virtual/opengl
+	alsa? ( media-libs/alsa-lib )
+	jack? ( media-sound/jack-audio-connection-kit )
+	nls? ( virtual/libintl )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )"
+
+S=${WORKDIR}/${PN}
+
+pkg_pretend() {
+	if has ccache ${FEATURES}; then
+		ewarn
+		ewarn "If you experience build failure, try turning off ccache in FEATURES."
+		ewarn
+	fi
+}
+
+src_prepare() {
+	default
+	eautoreconf
```

Comment:
The ebuild in this PR was intended to show good faith in the code quality from upstream for 0.9.44.1 version bump.

Not denying that I made a general assumption of previous issue(s) which needed patches had probably been fixed upstream, or the patch(es) probably addressed a runtime behavior in the first place rather than fixing a [un]documented build failure or other gentoo bug, or some other undocumented reason for the patch existing in earlier version.